relayer/config/config.go

```diff
@@ -6,7 +6,8 @@ type PolkadotConfig struct {
 }
 
 type ParachainConfig struct {
-	Endpoint string `mapstructure:"endpoint"`
+	Endpoint          string `mapstructure:"endpoint"`
+	KeyFetchBlockSize uint32 `mapstructure:"key-fetch-block-size"`
```

Comment:
I'm not sure its worth making this configurable. Its more a constant that is unlikely to change much.

Also for future reference, this `ParachainConfig` is purely for client and connection-related config, such as endpoints, timeouts, and so on. The struct is shared between three different relayers and so `KeyFetchBlockSize` will be irrelevent for the other relays (ethereum relay and the beefy relay). 

So if you want add a config key next time, rather put it into one of these structs: 

https://github.com/Snowfork/snowbridge/blob/2c61adf8e3eecf419ca238a5b6e6ba1cb6cde03c/relayer/relays/parachain/config.go#L10

https://github.com/Snowfork/snowbridge/blob/2c61adf8e3eecf419ca238a5b6e6ba1cb6cde03c/relayer/relays/parachain/config.go#L23

In the terminology I've used to organise configuration (for the parachain relay), A source is from where events originate (i.e the parachain and the relaychain), and the sink is where we need to write events (Ethereum).

The configuration for the other relays is modelled the same.

Reply:
Is it fine do leave this as a `const` in the code? Or should I move it to the appropriate Source/Sink Config?

Reply:
A const in the code should be fine